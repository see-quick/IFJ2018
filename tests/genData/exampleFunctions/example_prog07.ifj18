# Swap

a = 10
b = 30

print a, b, "\n"

temp = b

b = a
a = temp

print a, b,"\n"
