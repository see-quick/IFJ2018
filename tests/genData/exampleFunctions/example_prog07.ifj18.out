1030
3010
